src/OrbitGl/ThreadTrack.cpp

```diff
@@ -276,17 +276,17 @@ void ThreadTrack::Draw(GlCanvas* canvas, PickingMode picking_mode, float z_offse
   const float tracepoint_track_height = layout_->GetEventTrackHeight();
 
   if (!thread_state_bar_->IsEmpty()) {
-    thread_state_bar_->SetSize(canvas->GetWorldWidth(), thread_state_track_height);
+    thread_state_bar_->SetSize(canvas->GetViewport().GetWorldWidth(), thread_state_track_height);
```

Comment:
Nit: Extract the world_width into a local var.?

Reply:
Done